docs/reference/projectyml.rst

```diff
@@ -100,72 +113,111 @@ example of a dev config:
 .. code-block:: yaml
 
     pip:
-        index-url: https://devpi.example.com/dev/
+        index-url: https://my_repo.secure.company.com/repository/dev
         extra-index-url: []
         pre: true
         use-system-config: false
 
 2) In production
 
-Using a single pip index is still the recommended approach, and the use of pre-release versions should be disabled. Here is an
-example of a config suitable in production:
+Using a single pip index is still the recommended approach, and the use of pre-release versions should be disabled.
+
+
+If the pip configuration is fully managed at the system level and secure, use ``use-system-config: true``
+and add any potential project-specific repo that is not managed at the system level through the ``extra-index-url``
+option e.g.:
 
 .. code-block:: yaml
 
     pip:
-        index-url: https://devpi.example.com/stable/
-        extra-index-url: []
+        extra-index-url: [https://my_repo.secure.company.com/repository/project-production]
```

Comment:
I would keep the example to the simple case of "everything is managed at the system", so drop the extra index. There is already a section that explains that you can set it and what it does, so I really see no need for this, and I think it can be confusing for someone just looking for a config for production.

This may need a slight rephrasing of the paragraph above.